queries.sql

```diff
@@ -9,6 +9,53 @@ SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
```

Comment:
- [ ] As required in the project requirements kindly verify the changes that were made **and** persist after the commit so it would be nice if you follow this requirement and add `SELECT * FROM animals;` before the commit :+1: After making the change your query will like below :arrow_down:

```SQL
BEGIN;
UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
SELECT * FROM animals; -- verify that changes were made
COMMIT;
SELECT * FROM animals; -- verify that change persists after commit
```

Reply:
![Screenshot 2023-05-04 163544](https://user-images.githubusercontent.com/44966918/236225604-209cf166-89e7-4365-bf12-09e28a9454f8.png)